service/collector/release.go

```diff
@@ -0,0 +1,102 @@
+package collector
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/giantswarm/k8sclient/v3/pkg/k8sclient"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/prometheus/client_golang/prometheus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	gaugeValue float64 = 1
+)
+
+const (
+	namespace = "release_operator"
+	subsystem = "release"
+)
+
+var (
+	ReleaseDesc *prometheus.Desc = prometheus.NewDesc(
+		// TODO: What is the most descriptive name for this? Maybe change to "states" or "statuses"?
```

Comment:
How about naming it "releases" since it tells the name, the state (from spec), and the readiness (from status) of all releases? I can imagine adding metrics about other resources to the operator in the future.

Reply:
The current metric name when concatenated is "release_operator_release_info", so changing "info" to "releases" would yield "release_operator_release_releases" XD 

I think the fact that it's about releases is covered. "Info" is what most other operators are doing, but just seems very unspecific to me, so I was leaning "states". Then the full metric name would be "release_operator_release_states". Open to other thoughts

Reply:
I see what you mean. The only important part of the metric in my opinion is whether the release is satisfied or not, so I think `release_operator_release_status` would be a reasonable name.

Reply:
That sounds good, going with that